### PR TITLE
Feature/server setup

### PR DESCRIPTION
**Pull request by raghav-gkmit**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,6 @@
+SERVER_PORT = 4367
```
Comment (dhruvildave007) [inline .env]: why added env variables with credentials in git?
create `.env.sample` and just add this 
```
DB_HOST=
DB_USERNAME=
DB_PORT=
DB_DATABASE=
DB_PASSWORD=
```

also and `.env` in `.gitignore`

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (dhruvildave007) [inline .gitignore:1]: add .env file in .gitignore

File: config/config.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    host: "127.0.0.1",
+    dialect: "mysql",
+  },
+  production: {
+    username: "root",
+    password: null,
+    database: "database_production",
+    host: "127.0.0.1",
+    dialect: "mysql",
+  },
```
Comment (dhruvildave007) [inline config/config.js]: use environment variable from `.env` for production as well.

**Review by dhruvildave007 — CHANGES_REQUESTED** [review]
please work on required changes

File: config/config.js (hunk 3 above)
Comment (dhruvildave007) [reply config/config.js]: @raghvendragkmit not resolved yet

Comment (raghav-gkmit) [reply config/config.js]: resolved .env for production

**Review by dhruvildave007 — APPROVED** [review]

**Pull request merged**